src/topology/category/Top/limits.lean

```diff
@@ -43,6 +43,19 @@ def limit_cone (F : J ⥤ Top.{u}) : cone F :=
     naturality' := λ j j' f,
                    continuous_map.coe_inj ((types.limit_cone (F ⋙ forget)).π.naturality f) } }
 
+/--
+An alternative choice of limit cone for a functor `F : J ⥤ Top`
+This is useful when we want to explicitly express limits of topological spaces
+as subspaces of a cartesian product.
+-/
+def alt_limit_cone (F : J ⥤ Top.{u}) : cone F :=
+{ X := { α := { u : Π j : J, F.obj j | ∀ {i j : J} (f : i ⟶ j), F.map f (u i) = u j} },
```

Comment:
```suggestion
{ X := Top.of { u : Π j : J, F.obj j | ∀ {i j : J} (f : i ⟶ j), F.map f (u i) = u j} },
```
is I think the usual pattern

Reply:
I'm going to try to change `limit_cone` when I have some time later today, and I'll make sure to use `Top.of`.